src/v/raft/consensus.cc

```diff
@@ -258,6 +258,7 @@ void consensus::process_append_entries_reply(
   follower_req_seq seq_id) {
     auto is_success = update_follower_index(node, std::move(r), seq_id);
     if (is_success) {
+        maybe_update_majority_replicated_index();
```

Comment:
i think that the commit message should discuss how this relates (if it relates at all) to the raft dissertation discussion of support linearizable reads.

Reply:
It doesn't our raft support relaxed consistency replication for `acks=1`. This doesn't change the linearizable reads/writes semantics.